src/spetlr/autoloader/autoloader_handle.py

```diff
@@ -0,0 +1,62 @@
+from pyspark.sql import DataFrame
+
+from spetlr.configurator.configurator import Configurator
+from spetlr.delta.delta_handle import DeltaHandleInvalidFormat
+from spetlr.exceptions import SparkVersionNotSupportedForSpetlrStreaming
+from spetlr.spark import Spark
+from spetlr.tables import TableHandle
+
+
+class AutoloaderHandle(TableHandle):
+    def __init__(
+        self,
+        *,
+        location: str,
+        schema_location: str,
+        data_format: str,
+    ):
+        """
+        location: the location of the delta table
+        checkpoint_path: The location of the checkpoints, <table_name>/_checkpoints
+            The Delta Lake VACUUM function removes all files not managed by Delta Lake
+            but skips any directories that begin with _. You can safely store
+            checkpoints alongside other data and metadata for a Delta table
+            using a directory structure such as <table_name>/_checkpoints
+            See: https://docs.databricks.com/structured-streaming/delta-lake.html
+        data_format: the data format of the files that are read
+        """
+
+        if Spark.version() < Spark.DATABRICKS_RUNTIME_10_4:
+            raise SparkVersionNotSupportedForSpetlrStreaming()
+
+        self._location = location
+        self._data_format = data_format
+
+        self._schema_location = schema_location
+
+        self._validate()
+
+    @classmethod
+    def from_tc(cls, id: str) -> "AutoloaderHandle":
+        tc = Configurator()
+        return cls(
+            location=tc.table_property(id, "path", ""),
+            data_format=tc.table_property(id, "format", ""),
+            schema_location=tc.table_property(id, "schema_location", ""),
```

Comment:
here schema_location again. I understand checkpoint_path, but what is a schema_location?